docs/integrations/sources/woocommerce.md

```diff
@@ -0,0 +1,61 @@
+# Woocommerce
+
+## Sync overview
+
+The Woocommerce source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+
+This source can sync data for the [Woocommerce API](http://woocommerce.github.io/woocommerce-rest-api-docs).
+
+This Source Connector is based on a [Airbyte CDK](https://docs.airbyte.io/contributing-to-airbyte/python).
+
+### Output schema
+
+This Source is capable of syncing the following core Streams:
+
+* [Abandoned Checkouts](https://help.shopify.com/en/api/reference/orders/abandoned_checkouts)
+* [Collects](https://help.shopify.com/en/api/reference/products/collect)
+* [Custom Collections](https://help.shopify.com/en/api/reference/products/customcollection)
+* [Customers](https://help.shopify.com/en/api/reference/customers)
+* [Draft Orders](https://help.shopify.com/en/api/reference/orders/draftorder)
+* [Discount Codes](https://shopify.dev/docs/admin-api/rest/reference/discounts/discountcode)
+* [Metafields](https://help.shopify.com/en/api/reference/metafield)
+* [Orders](https://help.shopify.com/en/api/reference/orders)
+* [Products](https://help.shopify.com/en/api/reference/products)
+* [Transactions](https://help.shopify.com/en/api/reference/orders/transaction)
+* [Pages](https://help.shopify.com/en/api/reference/online-store/page)
+* [Price Rules](https://help.shopify.com/en/api/reference/discounts/pricerule)
+
+### Data type mapping
+
+| Integration Type | Airbyte Type | Notes |
+| :--- | :--- | :--- |
+| `string` | `string` |  |
+| `number` | `number` |  |
+| `array` | `array` |  |
+| `object` | `object` |  |
+
+### Features
+
+| Feature | Supported?\(Yes/No\) | Notes |
+| :--- | :--- | :--- |
+| Full Refresh Sync | Yes |  |
+| Incremental - Append Sync | Yes |  |
+| Namespaces | No |  |
```

Comment:
Maybe remove this part and add for each stream if it's possible to do incremental

Reply:
If possible yes.